app/views/settings/index.html.erb

```diff
@@ -139,6 +139,25 @@
       </div>
     <% end %>
 
+    <% if Keybase.enabled? %>
+      <div class="boxline">
+        <%= f.label :kb_username,
+          raw("<a href=\"https://keybase.io/\">Keybase</a>:"),
+          :class => "required" %>
+        <span>
+        <% if @edit_user.keybase_signatures.present? && Keybase.verify(@edit_user.keybase_signatures[0].kb_username, @edit_user.keybase_signatures[0].sig_hash, @edit_user.username) %>
+          Linked to
+          <strong><a href="https://keybase.io/<%= h(@edit_user.keybase_signatures[0].keybase_username)
```

Comment:
Please show all, not one.

Reply:
I'm not sure how to iterate with this template language, can you provide some more information

Reply:
```ruby
<% @edit_user.keybase_signatures.each do |kb_signature| %>
  <strong><a href="https://keybase.io/<%= h(kb_signature.keybase_username)</a></strong>
<% end %>